mist-common/src/main/java/edu/snu/mist/common/operators/StateTransitionOperator.java

```diff
@@ -40,104 +36,126 @@
  */
 public final class StateTransitionOperator extends OneStreamOperator implements StateHandler {
```

Comment:
Could you please address the code style issue separately?  It is hard to know which parts are actually related to this PR.

Reply:
Sorry for the inconvenience. I will create a separate issue.